build.sh

```diff
@@ -88,3 +88,17 @@ update_interop_2022() {
 
 mkdir -p out/data/interop-2022/
 update_interop_2022 out/data/interop-2022/
+
+update_interop_year() {
+  local YEAR="${1}"
+
+  mkdir -p out/data/interop-${YEAR}/
+  node interop-scoring/main.js --year=${YEAR} --to=${TO_DATE}
+  node interop-scoring/main.js --year=${YEAR} --to=${TO_DATE} --experimental
+
+  mv interop-${YEAR}-*.csv out/data/interop-${YEAR}/
+}
+
+update_interop_year 2021
+update_interop_year 2022
+update_interop_year 2023
```

Comment:
Interop 2021-2023 will be invoked with the new script and used in the interop dashboard redesign. The old script will also still run until removed.